ServiceRequest/ServiceRequest-WGSTestOrderFormUpdated-DirectToLab-Example.json

```diff
@@ -122,7 +122,7 @@
   ],
   "relevantHistory": [
     {
-      "reference": "Provenance/Provenance-ServiceRequest-WGSTestOrderForm-DirectToLab-Example"
+      "reference": "Provenance/Provenance-Provenance-WGSTestOrderForm-DirectToLab-Example"
```

Comment:
Duplicated 'Provenance' in name

Reply:
Not a duplicate but a name change to Provenance-Provenance-WGSTestOrderForm-DirectToLab-Example

Reply:
Hi Kazeem, on the files changed, the filename is Provenance/Provenance-WGSTestOrderForm-DirectToLab-Example.json (1 Provenance in the path, and 1 in the file name), the reference has 3

Reply:
Now sorted as expected.